tests/test_generic.py

```diff
@@ -41,6 +41,8 @@ def test_tor_updater(self):
         db_file_path = tor.Tor.updater()
         self.assertTrue(os.path.exists(db_file_path))
 
+
+class CronTestsYara(TestCase):
```

Comment:
I understand why you made this change but I'd say for now we revert this and implement a proper solution later that also covers the tor updater, maxmind updater, etc. methods.

Reply:
My main focus in this PR is to have a more proper coverage and management of the Docker Analyzers. I think that, with this new version, it is at least the third time that we make a release and the docker analyzers do not work properly.

So, I took that opportunity to rework the PR automation and also trying to separate cases between master and develop to speed up the CI.

This is a fast fix to have the tests work correctly with the REPO_DOWNLOADER_ENABLED=False enabled that would save us time and will avoid the abuse of that external service. This is extremely important to fix

Reply:
anyway I have removed that specific part (REPO_DOWNLOADER_ENABLED) because it would have failed too many analyzers.